katas/langtonant/matias/main.test.js

```diff
@@ -0,0 +1,6 @@
+import { Ant } from './main.js';
```

Comment:
This is a good start. Please add more tests for verifying all the functionality you have here